client/network-manager.go

```diff
@@ -806,7 +817,7 @@ func (c *NetworkManagerClient) Points(nodeID string, points []data.Point) {
 
 // EdgePoints is called when the client's node edge points are updated
 func (c *NetworkManagerClient) EdgePoints(
-	nodeID string, parentID string, points []data.Point,
+	_ string, _ string, _ []data.Point,
```

Comment:
I'm not a huge fan of this linter suggestion, but it's also fine. I think there's some value in keeping the variable names in case this method becomes useful in the future.

Reply:
yeah, I can go either way on this -- it is easy to look up the params, and knowing when you are not using a param is valuable too. I'll leave as is for now so we can get pass CI.